### PR TITLE
Manipulate Multiple Enemy NPCs

### PR DESCRIPTION
**Pull request by ohbeanz**
Potential fix for Issue #1326

### CODE REVIEW

File: combat-trainer.lic [hunk 1]
```diff
@@ -1102,7 +1102,19 @@ class ManipulateProcess
 
   def manipulate
     bput('manip stop all', 'You relax your will', 'But you aren')
-    bput("manipulate friendship #{@filtered_npcs.first}",  'You attempt to empathically manipulate', 'You strain')
+    @manipulate_count = 0
```
Comment (rpherbig) [inline combat-trainer.lic]: Should this be a local variable (i.e. drop the `@`)? I don't see it used outside of this method...

File: combat-trainer.lic [hunk 2]
```diff
@@ -1102,7 +1102,19 @@ class ManipulateProcess
...
+
+    @filtered_npcs.each_with_index { |npc, index|
```
Comment (rpherbig) [inline combat-trainer.lic]: Our convention for multiple-line blocks is to use `do`/`end` instead of `{}`

File: combat-trainer.lic [hunk 3]
```diff
@@ -1102,7 +1102,19 @@ class ManipulateProcess
...
+      if @manipulate_count >= @threshold
```
Comment (rpherbig) [inline combat-trainer.lic]: You can make this a one-liner: `break if @manipulate_count >= @threshold` (or `break unless @manipulate_count < @threshold` if you think that reads better).

File: combat-trainer.lic [hunk 4]
```diff
@@ -1102,7 +1102,19 @@ class ManipulateProcess
...
+        break
+      end
+      
+      ordinal_string = $ORDINALS[index]
+	  
+      bput("manipulate friendship #{ordinal_string} #{npc}", 'You\'re already manipulating', 'You attempt to empathically manipulate', 'You strain')
+	  @manipulate_count += 1;
```
Comment (rpherbig) [inline combat-trainer.lic]: This line is indented too far.

File: spinner.lic [hunk 5]
```diff
@@ -92,4 +92,4 @@ class Spinnerette
 end
 
 # Call this last to avoid the need for forward declarations
-Spinnerette.new
+Spinnerette.new
```
Comment (rpherbig) [inline spinner.lic]: Could you revert this change?

File: combat-trainer.lic (hunk 1 above)
Comment (ohbeanz) [inline combat-trainer.lic]: Agreed.
